cmd/crictl/image.go

```diff
@@ -147,7 +147,7 @@ var listImageCommand = &cli.Command{
 		&cli.StringSliceFlag{
 			Name:    "filter",
 			Aliases: []string{"f"},
-			Usage:   "The filtering flag format is of 'dangling=(true/false)', 'reference=regex', '(before|since)=<image-name>[:<tag>]|<image id>|<image@digest>'",
+			Usage:   "Filter output based on provided conditions.\nEach filter has an AND style relationship with the other others.\nAvailable filters: \n* dangling=BOOL\n* reference=/Regular Expression/\n* before=<image-name>[:<tag>]|<image id>|<image@digest>\n* since=<image-name>[:<tag>]|<image id>|<image@digest>",
```

Comment:
Perhaps?

```suggestion
			Usage:   "Filter output based on provided conditions.\nAvailable filters: \n* dangling=(boolean - true or false)\n* reference=/regular expression/\n* before=<image-name>[:<tag>]|<image id>|<image@digest>\n* since=<image-name>[:<tag>]|<image id>|<image@digest>\nMultiple filters can be combined together.",
```

Produces:

```console
   --filter value, -f value [ --filter value, -f value ]  Filter output based on provided conditions.
      Available filters: 
      * dangling=(boolean - true or false)
      * reference=/regular expression/
      * before=<image-name>[:<tag>]|<image id>|<image@digest>
      * since=<image-name>[:<tag>]|<image id>|<image@digest>
      Multiple filters can be combined together.
```

I wish the CLI package aligned usage to the right nicely.